app/models/proxy.rb

```diff
@@ -278,15 +278,16 @@ def publish_events
     nil
   end
 
+  # TODO: remove unused method, there is no such attribute
```

Comment:
Is it used here?
https://github.com/3scale/porta/blob/de8c865c1022ca7d6b981fdb9bae14e2806def03/app/models/service.rb#L494